potofolio.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css"> 
+    <title>Document</title>
+</head>
+<body>
+    <div id="header">
+        <div id="logoplaceholder"></div>
+        <button class="menu">
+            <span class="line"></span>
+            <span class="line"></span>
+            <span class="line"></span>
+         </button>
+      
+    </div>
+    <div id="container" >
+        
+        <div id="aboutme" class="intro">
+            <h1 >Hey there.
+            I’m Alex.</h1>
+           <h1 class="job"> I’m a software developer</h1>
+           <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.</p>
+        </div>
+        <div class="icon-bar">
+            <i><a href="https://www.twitter.com">
+                <img src="twitter.png">
+            </a></i>
+            <i> <a href="https://www.Linkedin.com">
+                <img src="Linkedin icon.png">
+           </a></i>
+           <i> <a href="https://www.Medium.com">
+            <img src="medium.png">
+            </a></i>
+            <i> <a href="https://www.github.com">
+            <img src="github.png">
+            </a></i>
+            <i> <a href="#">
+            <img src="angellist.png">
+            </a></i>
+        </div>
```

Comment:
- [ ] For this section, kindly make the list of icons a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).
- Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.